Comment-Code-Poltergeists/nutshell
# Sp articles littlethings

# Description

fixed little things

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

git fetch
gc sp-articles-littlethings
check that the page is not super long/tall
tell me if you're ok with the font



# Checklist:
